### PR TITLE
Updated main network to nn-83a0d6daf7e5.nnue

### PR DESCRIPTION
**Pull request by TonyCongqianWang**
Passed STC:
https://tests.stockfishchess.org/tests/live_elo/69f902413a3c3e525bb2b465
LLR: 2.94 (-2.94,2.94) <0.00,2.00>
Total: 27264 W: 7199 L: 6895 D: 13170
Ptnml(0-2): 80, 3139, 6903, 3417, 93

Passed LTC:
https://tests.stockfishchess.org/tests/live_elo/69f98b103a3c3e525bb2b4f7
LLR: 2.94 (-2.94,2.94) <0.50,2.50>
Total: 32352 W: 8509 L: 8200 D: 15643
Ptnml(0-2): 10, 3439, 8979, 3728, 20

Nettest Run
https://gitlab.com/cscs-ci/ci-testing/webhook-ci/mirrors/5137461961076608/2926829081096545/-/pipelines/2496280249

Nettest PR:
https://github.com/vondele/nettest/pull/329

Makes quite a few changes to the trainer. In particular recipe does not use a lambda (game score + outcome blending parameter) schedule for the most part.

Accidentally dropped loss weighing parameters. Unclear how big the effect of that is

### CODE REVIEW

**Comment by xu-shawn** [comment]
Could you summarize what was changed in the training of this network? It's rather hard to sift through multiple diffs

**Comment by TonyCongqianWang** [comment]
So as a preamble: It is kinda still WIP unfortunately. I tried a lot of things and it's unclear yet which things truly matter...

The recipe is can still be tuned. Quite a bit probably.

- The ranger21 optimizer was replaced by an roughly equivalent one (with small bugfixed) which is faster.

- Hyperparameters where added and tuned (lr and lambda). Additionally LR now uses a consine scheduling instead of gamma. Parameters of the schedule where also tuned (by hand).

- Most impactful change is probably the removal of a lambda schedule. Instead of starting lambda high and decreasing it over the run, the average lambda stays constant, but a random term is added on top. efficient_swa2 and efficient_swa3 runs only differed in the random term strengths and that made a big difference in perform 

- Accidentally removed w1 and w2 Hyperparameters. Which removes loss weighing. No idea yet what effect that has on it's own (it's being tested right now).

- Stage lengths where changed it's now 250+300+550+1050+2050=4200. Though this is still somewhat experimental and it's not sure what the best approach is. The upside of this was to get earlier feedback and the elo progression was at least stable across two slightly different hyperparameter configs.

- ~~Added stochastic weight averaging, but that turns out to be actual bugged and probably not doing anything -- yet~~ After testing, it definitely does do something, although it's not entirely clear which part is bugged

**Comment by anematode** [comment]
(Cross-posting from Discord) For the record – although the previous net had a significant speedup as measured by vondele, Torom and me (~1% or so), this one is stable

```
efficient_swa_3: Nodes/second               : 1645209
master: Nodes/second               : 1645198
```

so we can be pretty confident that the strength improvement is from improved eval quality rather than higher sparsity
